packages/account/src/Components/currency-selector/__tests__/currency-selector.spec.tsx

```diff
@@ -26,8 +26,8 @@ describe('<CurrencySelector/>', () => {
                 is_virtual: 1,
                 landing_company_shortcode: 'virtual',
                 trading: {},
-                token: 'a1-sLGGrhfYPkeEprxEop2T591cLKbuN',
-                email: 'test+qw@deriv.com',
+                token: '',
```

Comment:
You have 5 codesmells in your PR. You can forget about Cognitive Complexities but others need to get fixed.